### DISCUSSION TITLE
No such variant in enum MultiAddress

### DISCUSSION THREAD

**Discussion by comien** [ParityAsia/substrate-newsletter#17] [Substrate/Polkadot 技术问题]
#### contracts.putCode 1002: Verification Error: Execution: Could not convert parameter `tx` between node and runtime: No such variant in enum MultiAddress: RuntimeApi, Execution: Could not convert parameter `tx` between node and runtime: No such variant in enum MultiAddress
🙏请问一下各位老师，上传合约时，这种错误该怎么解决呢？

**Comment by jimmychu0807** [comment]
你是用哪一個前端？你 Substrate 請更新到 v2.0.1 或 v3.0.0 吧
